more/associations/mt-summit.md

```diff
@@ -0,0 +1,23 @@
+---
+grand_parent: More
+parent: Associations
```

Comment:
Maybe we can do...

`parent: Events`

...and then, in `events.md`, add a section before the first table in which we have filtered content, like `AMTA`, `MT Summit`, `EAMT`, `WMT`, and `AAMT`?